package.json

```diff
@@ -25,6 +25,13 @@
     "typescript-eslint": "^7.10.0",
     "wireit": "^0.14.4"
   },
+  "pnpm": {
+    "ignoredOptionalDependencies": [
```

Comment:
See the [PNPM documentation](https://pnpm.io/package_json#pnpmignoredoptionaldependencies) on this field for more information.